nuget.config

```diff
@@ -2,8 +2,11 @@
 <configuration>
   <packageSources>
     <clear />
-    <add key="NuGet official package source" value="https://api.nuget.org/v3/index.json" />
+
+    <!-- do not add new sources to this file. If new packages are required, push them to one of these feeds. -->
     <add key="dotnetfeed" value="https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json" />
+    <add key="MUX-Dependencies" value="https://pkgs.dev.azure.com/ms/microsoft-ui-xaml/_packaging/MUX-Dependencies/nuget/v3/index.json" />
+
     <add key="localpackages" value="localpackages" />
```

Comment:
remind me what this was for? Do we still need it?

Reply:
This allows you to test nupkg files locally without pushing them to a feed. You can just put the nupkg file in the 'localpackages' dir.